numerics/polynomial_body.hpp

```diff
@@ -432,6 +418,32 @@ operator PolynomialInMonomialBasis<Value_, Argument_, higher_degree_,
   return Result(higher_coefficients, origin_);
 }
 
+template<typename Value_, typename Argument_, int degree_,
+         template<typename, typename, int> typename Evaluator>
+Value_
+PolynomialInMonomialBasis<Value_, Argument_, degree_, Evaluator>::operator()(
+    Argument const& argument) const {
+  return UseHardwareFMA
+             ? Evaluator<Value, Difference<Argument>, degree_>::Evaluate<
```

Comment:
Give a name to `Evaluator<...>` and use it in both branch to reduce the visual noise.  (Several places.)